tsconfig.base.json

```diff
@@ -47,6 +47,10 @@
       ],
       "@energinet-datahub/dh/core/routing": ["libs/dh/core/routing/src/index.ts"],
       "@energinet-datahub/dh/core/shell": ["libs/dh/core/shell/src/index.ts"],
+      "@energinet-datahub/dh/dev-examples": ["libs/dh/dev-examples/src/index.ts"],
+      "@energinet-datahub/dh/dev-examples/feature-processes": [
+        "libs/dh/dev-examples/feature-processes/src/index.ts"
+      ],
```

Comment:
issue: this way of nesting one library inside another diverges slightly from the rest of the codebase. I'm mostly concerned about config files where a file from the "parent" lib "leaks" into the nested lib. Why not separate the two libs since the "boilerplate" files are already there?

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3958/commits/7b43583eda02f31a3dcc965e51f79f73dcbe2ddb